source/Game_Setup.py

```diff
@@ -0,0 +1,44 @@
+from source.NewCardGames import *
+
+class GameSetup:
+    def __init__(self):
+        #Initialize the objects that need to be initialized
+        self.cards_in_play = []
+        #Evolve this into creating a Fugitive object and setting object's deck to starting_cards
+        self.fugitive_deck = starting_cards
```

Comment:
Dude, this looks great! Great use of comments as well.

The only thing I am worried about is the self.fugitive_deck pulling from a global variable. When we change the NewCardGames.py to just CardGames.py, you will not be able to globally call it. We need to figure out a way to pull the starting/escape cards from the split_deck function.

Reply:
Thank you brother! Since we are migrating this class to the CardGames file, I will use the split_deck function and access the starting_cards via index from the tuple that the function returns.